Setup64/Setup64.vdproj

```diff
@@ -662,7 +662,7 @@
             {
             "AssemblyRegister" = "3:1"
             "AssemblyIsInGAC" = "11:FALSE"
-            "AssemblyAsmDisplayName" = "8:WasatchNET, Version=2.4.16.0, Culture=neutral, processorArchitecture=AMD64"
+            "AssemblyAsmDisplayName" = "8:WasatchNET, Version=2.4.10.0, Culture=neutral, processorArchitecture=AMD64"

```

Comment:
Guess I expected 2.5?

Reply:
as would I, but for the purposes of these serial PRs I'm not too worried about it

Reply:
it's probably due to merging main back into this branch where main is "ahead," leading to discrepancies